source/mainwindow.cpp

```diff
@@ -272,33 +279,87 @@ void MainWindow::on_DepartmentButton_clicked()
-        ui->inputFrame->setStyleSheet("QLineEdit{border-width:2px; color: gainsboro; background-color: black};");
+        ui->inputFrame->setStyleSheet("");
         ui->menuBar->setStyleSheet("background: dimgrey; border-style: outset; border-color: dimgrey; color: gainsboro;");
-        ui->menuView->setStyleSheet("background: black;");
+        ui->menuView->setStyleSheet("background-color: black;");
+        ui->scheduleTable->setStyleSheet("QTableWidget{border-color: dimgrey; border-width: 2px;}"
+                                         "QHeaderView::section{background-color: black; color: gainsboro;}"
+                                         "QTableCornerButton::section{background-color: black; border-color: dimgray; border-width: 2px;}"
+                                         "QTableWidget {alternate-background-color: dimgrey; gridline-color: white;}");
 
         darkMode = true;
 
     } else {
 
-        ui->centralWidget->setStyleSheet("background: white; border-style: outset; border-color: black; color: black;");
+        ui->centralWidget->setStyleSheet("QWidget{background: white; border-style: outset; border-color: black; color: black;}"
+                                         "QLineEdit{border-width:1px; color: black; background-color: white};");
         ui->scrollAreaWidgetContents->setStyleSheet("QPushButton{border-width: 1px; color: black; background-color: white;}"
                                                     "QPushButton::Pressed{border-width: 2px; color: gainsboro; background-color: black}"
                                                     "QPlainTextEdit{border-width: 1px;}");
-        ui->inputFrame->setStyleSheet("QLineEdit{border-width:1px; color: black; background-color: white};");
+        ui->inputFrame->setStyleSheet("");
         ui->menuBar->setStyleSheet("background: white; border-style: outset; border-color: black; color: black;");
         ui->menuView->setStyleSheet("background: white;");
+        ui->scheduleTable->setStyleSheet("QTableWidget{border-color: black; border-width: 1px;}"
+                                         "QHeaderView::section{background-color: white; color: black;}"
+                                         "QTableCornerButton::section{background-color: white; border-color: black; border-width: 1px;}"
+                                         "QTableWidget {alternate-background-color: lightgrey; gridline-color: black;}");
 
         darkMode = false;
     }
 
 }
 
+
+void MainWindow::initialize_table(int rows)
+{
+
+    const int columns = 17;
+
+    auto table = ui->scheduleTable;
+
+    table->setRowCount(rows); //row number will be provided dynamically based on how many departments are scheduled
+
+    table->setColumnCount(columns);
+
+    QString headerRow[columns] = {"Section Type", "CRN", "Course", "Title", "Credit", "Max Enrollment", "Enrolled", "Available", "Wait List", "MTYP", "Days", "Start", "End", "Building", "Room", "Instructor", "Conflict"};
```

Comment:
Enrolled through MTYP don't need to be there as these columns will always be empty. The appropriate changes have already been made to the example output spreadsheet.